src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -80,7 +79,7 @@ public string LastName
             {
                 if (PlayerValidation.ValidateBirthYear(value))
                 {
-                    throw new ArgumentException(Resources.ValidationPlayerBirthYear, "BirthYear");
+                    throw new ArgumentException(Resources.ValidationPlayerBirthYear, nameof(Player.BirthYear));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The parameter name 'BirthYear' is not declared in the argument list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3928)